build_tools/update_environments_and_lock_files_pr.py

```diff
@@ -0,0 +1,47 @@
+import argparse
+import os
+import shlex
+import subprocess
+
+
+def execute_command(command):
+    command_list = shlex.split(command)
+    subprocess.run(command_list, check=True, text=True)
+
+
+def main():
+    comment = os.environ["COMMENT"].splitlines()[0].strip()
+
+    # Extract the command-line arguments from the comment
+    prefix = "@scikit-learn-bot update lock-files"
+    assert comment.startswith(prefix)
+    args_string = comment[len(prefix) :]  # including any leading spaces
+
+    # Extract the tag from the args (if any)
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--select-tag", default=None)
```

Comment:
I think it would be great to allow to pass specific marker, e.g. via a `--commit-marker` option?

I can imagine that one of the use case is when you want to update only the `doc` and `doc-min-dependencies` lock-files in which case you would use something like this:
```
@scikit-learn-bot please update lock-file --select-build doc --commit-marker '[doc build]` 
```